src/configs/battery.rs

```diff
@@ -23,17 +24,42 @@ impl<'a> RootModuleConfig<'a> for BatteryConfig<'a> {
             unknown_symbol: "",
             empty_symbol: "",
             format: "[$symbol$percentage]($style) ",
-            display: vec![BatteryDisplayConfig {
-                threshold: 10,
-                style: "red bold",
-            }],
+            display: vec![BatteryDisplayConfig::new()],
             disabled: false,
         }
     }
 }
 
-#[derive(Clone, ModuleConfig)]
+#[derive(Clone)]
 pub struct BatteryDisplayConfig<'a> {
     pub threshold: i64,
     pub style: &'a str,
+    pub symbol: &'a str,
```

Comment:
I think `Option<'a str>` would work better for this.

Reply:
Thank you for your comments.
I see. I will try to change it.

Reply:
I changed to `Option<'a str>`.